markdownlint/docs/README.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdownlint-disable-next-line -->
```

Comment:
I wasn't aware about them being ignored by the linter action, so I added those disable comments to prevent the local linter extension flagging the opening header for not being a level 3 header (https://github.com/TheOdinProject/curriculum/pull/27674/commits/8c2a79a06da0459972d226972e6d645a582b2dd0)

```json
    // first-line-heading/first-line-h1
    // Enforces the first heading on a lesson page to be an H3, since the page template renders an H1 and H2 automatically
    "MD041": {
      "level": 3
    },
```
But if this is a non-issue due to the lint actions, then I can just remove disable comments and leave them as they were.

Reply:
These files _shouldn't_ get flagged, but we can always see if you remove them the disable comments

Reply:
Looks like it's all good then to not have the disable comments :+1: